src/GafferSceneUIModule/RenderPassEditorBinding.cpp

```diff
@@ -315,12 +318,17 @@ class RenderPassPath : public Gaffer::Path
 					return new StringData( names().back().string() );
 				}
 			}
-			else if( name == g_renderPassEnabledPropertyName )
+			else if( name == g_renderPassEnabledPropertyName || name == g_renderPassEnabledWithoutAdaptorsPropertyName )
 			{
 				const PathMatcher p = pathMatcher( canceller );
 				if( p.match( names() ) & PathMatcher::ExactMatch )
 				{
 					Context::EditableScope scopedContext( getContext() );
+					if( name == g_renderPassEnabledWithoutAdaptorsPropertyName )
+					{
+						const bool disableAdaptors = true;
+						scopedContext.set<bool>( g_disableAdaptorsContextName, &disableAdaptors );
```

Comment:
Could I make a case for having an `enableAdaptors` context variable instead?

- We generally try to avoid inverted "true means off" semantics.
- I think adaptors being off by default when using `RenderPassPath::context()` directly probably makes the most sense - because RenderPasses provide the "recipe" for adaptors, we want to see that recipe rather than the end result in the RenderPassEditor. If I made an adaptor that monkied with `renderPass:inclusions`, I don't think that should be reflected in the RenderPassEditor, but I think it would be currently.

Reply:
I also wonder if the context monkeying might be better off in the RenderPassNameColumn, so RenderPassPath is a bit conceptually purer, and the adaptor juggling is in higher-level parts of the UI. I suppose RenderPassPath is already a bit UI-focussed what with the grouping options, but it still feels like maybe it'd be nice to keep it closer to just a raw Path-based view onto a single scene.